test/system/incomplete_incidents_test.rb

```diff
@@ -50,7 +50,5 @@ class IncompleteIncidentsTest < ApplicationSystemTestCase
     visit incomplete_incidents_url
 
     click_button 'Print'
-
-    assert_equal find('embed')['type'], 'application/pdf'
```

Comment:
This doesn't work in Firefox. But since the test really just is there to make sure no internal server errors occur from clicking on 'Print', this is sufficient (I verified this by intentionally breaking the PDF, and the test failed).